test/heartbeat/heartbeat.js

```diff
@@ -63,9 +64,10 @@ const generator2 = {
   subjects: [{ absolutePath: 'S1.S2', name: 'S2' }],
   generatorTemplate: {
     connection: {
-      url: 'http://www.abc.com',
+      url: refocusUrl,
```

Comment:
the SGT's connection url is the url of some remote data source -- NOT the url of the refocus instance